readme.md

```diff
@@ -30,6 +30,8 @@ Tailwind CSS is a utility-first CSS framework for rapidly building custom user i
 - [Tailwind Themes](https://www.tailwind-themes.com)
 - [Tailwind Cheat Sheet](http://nerdcave.com/tailwind-cheat-sheet)
 - [Alfred Workflow - Search Tailwind Docs with Alfred](https://github.com/clnt/alfred-tailwindcss-docs)
+- [ng-tailwindcss - The happy marriage of Tailwind with Angular CLI](https://www.npmjs.com/package/ng-tailwindcss)
```

Comment:
Can you link to the github repo over the npm page.

Reply:
Done. :)